### ISSUE TITLE
Gnome 43 - Nautilus - No Coloured Sidebar or Highlight

### ISSUE THREAD

**Issue by tiefling** [vinceliuice/Orchis-theme#256]
Under Gnome 43, Nautilus doesn't show any colours in the Sidebar or Highlight anymore when Orchis is enabled (Tried Dark, Light and Standard types).

**Comment by vinceliuice** [comment]
Run `./install.sh -l`

**Comment by meaz** [comment]
Problem with that is that you can't switch from Dark to Light / light to dark. I mean you can, but Nautilus theme doesn't change.

**Comment by razcore-rad** [comment]
That's true, we can't easily select the theme and variants. Instead you can do it manually for now. After you install the themes you can do:

```
cd .config/gtk-4.0
ln -sf ../../.themes/Orchis-Dark-Compact/gtk-4.0/* .
```

And that should fix it.

![image](https://user-images.githubusercontent.com/1177508/204736477-f611c57c-9d43-495f-91cb-993c921bba01.png)

**Comment by meaz** [comment]
yes, but what I meant, is that once you have that done, then if you use the Dark mode switch button to turn if off, it would not turn it off for nautilus.

**Comment by vinceliuice** [comment]
This is not a perfect way to theme libadwaita, as you said once this have done, Dark mode switch will not work, and for now there is no fix for this

**Comment by Demetrio92** [comment]
FWIW you can use a fork of nautilius: https://github.com/EliverLara/Nordic/issues/221#issuecomment-1336289718
